modules/osdk-run-locally.adoc

```diff
@@ -64,7 +69,104 @@ ifdef::helm[]
 {"level":"info","ts":1612652420.2309358,"logger":"controller-runtime.manager.controller.nginx-controller","msg":"Starting workers","worker count":8}
 ----
 endif::[]
+ifdef::java[]
+. Run the following command to compile the Operator:
++
+[source,terminal]
+----
+$ mvn clean install
+----
++
+.Example output
+[source,terminal]
+----
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  11.193 s
+[INFO] Finished at: 2021-05-26T12:16:54-04:00
+[INFO] ------------------------------------------------------------------------
+----
+
+. Run the following command to install the CRD to the default namespace:
++
+[source,terminal]
+----
+$ oc apply -f target/kubernetes/memcacheds.cache.example.com-v1.yml
+----
++
+.Example output
+[source,terminal]
+----
+customresourcedefinition.apiextensions.k8s.io/memcacheds.cache.example.com created
+----
+
+. Create a file called `rbac.yaml` as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: memcached-operator-admin
+subjects:
+- kind: ServiceAccount
+  name: memcached-quarkus-operator-operator
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+----
+
+. Run the following command to grant `cluster-admin` privileges to the `memcached-quarkus-operator-operator` by applying the `rbac.yaml` file:
++
+[source,terminal]
+----
+$ oc apply -f rbac.yaml
+----
+
+. Enter the following command to run the Operator:
++
+[source,terminal]
+----
+$ java -jar target/quarkus-app/quarkus-run.jar
+----
++
+[NOTE]
+====
+The above `java` command will run the Operator and remain running until you kill it. You will need another terminal to complete the rest of these commands.
```

Comment:
> ! above adjective, adverb, preposition
> 2. Do not use to indicate a relative location in a document, as in "the above restrictions" or "the steps above". 

> ! kill verb
Do not use except when you are documenting a UNIX system, such as AIX. Use "end" or "stop".